src/ctr_logging.c

```diff
@@ -134,9 +137,10 @@ void configure_log_drivers(gchar **log_drivers, int64_t log_size_max_, char *cuu
  * parse_log_path branches on log driver type the user inputted.
  * log_config will either be a ':' delimited string containing:
  * <DRIVER_NAME>:<PATH_NAME> or <PATH_NAME>
- * in the case of no colon, the driver will be kubernetes-log-file,
+ * in the case the log driver is 'k8s-stream-file', the <PATH_NAME> must be present.
+ * in the case of no colon, the driver will be k8s-file,
  * in the case the log driver is 'journald', the <PATH_NAME> is ignored.
- * exits with error if <DRIVER_NAME> isn't 'journald' or 'kubernetes-log-file'
+ * exits with error if <DRIVER_NAME> isn't 'journald' or 'k8s-file'
```

Comment:
nit, we should probably add `k8s-stream-file` to this list

Reply:
Fixed.